samples/matter/thermostat/src/temperature_manager.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+
+#include "temperature_manager.h"
+#include "app_config.h"
+#include "app_task.h"
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+using namespace chip;
+using namespace ::chip::DeviceLayer;
+
+constexpr EndpointId kThermostatEndpoint = 1;
+
+namespace ThermostatAttributes = chip::app::Clusters::Thermostat::Attributes;
+
+
+CHIP_ERROR TemperatureManager::Init()
+{
+    app::DataModel::Nullable<int16_t> temp;
+    int16_t heatingSetpoint, coolingSetpoint;
```

Comment:
seems redundant, we can write these values directly to the class members, e.g. `&mHeatingCelsiusSetPoint`